debian/bareos-filedaemon-mariabackup-python-plugin.install

```diff
@@ -0,0 +1,2 @@
+/usr/local/lib64/bareos/plugins/bareos-fd-mariabackup.py*
```

Comment:
/usr/local seems wong here ? Mean the file shouldn't be part of the tree....

Reply:
removed